### PR TITLE
Hypervisor manager

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[Added resources file for HypervisorManager supporting API800, API1000, API1200]

### Issues Resolved
[]

### Check List
- [X] New functionality includes testing.
  - [X] All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [X] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [X] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - Enclosure Group
 - FC Network
 - FCoE Network
+- Hypervisor Managers
```
Comment (AnushaY1916) [inline CHANGELOG.md:16]: Please keep this resource under an unreleased version. Don't try to update the notes of a released version

Comment (AnushaY1916) [reply CHANGELOG.md:16]: In the PR description, under issues, you have put a link to a PR. Is that PR related to this? Recheck it once.
And under issues, links to issues must be given but not a PR link.

File: examples/hypervisor_manager.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	"github.com/HewlettPackard/oneview-golang/utils"
+	"os"
+)
+
+func main() {
+	var (
+		ClientOV                        *ov.OVClient
+		hypervisor_manager              = "172.18.13.11"
+		hypervisor_manager_display_name = "HM2"
+		//hypervisor_manager_2 = "eth88"
```
Comment (jyothis-gireesan) [inline examples/hypervisor_manager.go:15]: Please remove the commented out code or add a statement on why it is commented

File: examples/hypervisor_manager.go [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+		os.Getenv("ONEVIEW_OV_PASSWORD"),
+		os.Getenv("ONEVIEW_OV_DOMAIN"),
+		os.Getenv("ONEVIEW_OV_ENDPOINT"),
+		false,
+		800,
+		"")
+	ovVer, _ := ovc.GetAPIVersion()
+	fmt.Println(ovVer)
+	initialScopeUris := &[]utils.Nstring{utils.NewNstring("/rest/scopes/03beb5a0-bf48-4c43-94a5-74b7b5de1255")}
+	hypervisorManager := ov.HypervisorManager{DisplayName: "HM1", Name: "172.18.13.11", Username: "dcs", Password: "dcs", Port: 443, InitialScopeUris: *initialScopeUris, Type: "HypervisorManagerV2"}
```
Comment (jyothis-gireesan) [inline examples/hypervisor_manager.go:28]: Intend the initialization line by line so it's readable for the customers.

File: ov/hypervisor_manager.go [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+	}
+
+	log.Debugf("GetHypervisorManagers %s", data)
+	if err := json.Unmarshal([]byte(data), &hypervisorManagers); err != nil {
+		return hypervisorManagers, err
+	}
+	return hypervisorManagers, nil
+}
+
+func (c *OVClient) AddHypervisorManager(hypM HypervisorManager) error {
```
[the remark this answers is not in the document]
Comment (nabhajit-ray) [reply ov/hypervisor_manager.go]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (nabhajit-ray) [reply CHANGELOG.md:22]: Done

File: examples/hypervisor_manager.go (hunk 3 above)
Comment (nabhajit-ray) [reply examples/hypervisor_manager.go]: Donr

File: CHANGELOG.md [hunk 5]
```diff
@@ -13,6 +19,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
...
```
Comment (jyothis-gireesan) [inline CHANGELOG.md:22]: Remove the addition here.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,12 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+# [v1.3.0](unreleased)
+### Notes
+- This release supportsi for API V800,V1000 and V1200 for HypervisorManager resource
+
+### Oneview Features supported
+- HypervisorManager 
 # [v1.2.0] (2019-12-13)
```
Comment (jyothis-gireesan) [inline CHANGELOG.md:10]: Add space between the version notes

File: examples/hypervisor_manager.go [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+	initialScopeUris := &[]utils.Nstring{utils.NewNstring("/rest/scopes/03beb5a0-bf48-4c43-94a5-74b7b5de1255")}
+	hypervisorManager := ov.HypervisorManager{DisplayName: "HM1",
+		Name:             "172.18.13.11",
+		Username:         "dcs",
+		Password:         "dcs",
+		Port:             443,
+		InitialScopeUris: *initialScopeUris,
+		Type:             "HypervisorManagerV2"}
+
+	err := ovc.AddHypervisorManager(hypervisorManager)
```
Comment (AnushaY1916) [inline examples/hypervisor_manager.go:33]: Rename the function name  here as well.

Comment (nabhajit-ray) [reply examples/hypervisor_manager.go]: Done

**Pull request merged**